src/ring_theory/noetherian.lean

```diff
@@ -184,6 +184,16 @@ fg_def.2 ⟨linear_map.inl R M P '' tb ∪ linear_map.inr R M P '' tc,
   (htb.1.image _).union (htc.1.image _),
   by rw [linear_map.span_inl_union_inr, htb.2, htc.2]⟩
 
+theorem fg_pi {ι : Type*} [fintype ι] {p : ι → submodule R M} (hsb : ∀ i, (p i).fg) :
```

Comment:
```
theorem fg_pi {ι : Type*} [fintype ι] {φ : ι → Type*} [Π (i : ι), add_comm_monoid (φ i)]
  [Π (i : ι), module R (φ i)] {p : Π (i : ι), submodule R (φ i)} (hsb : ∀ i, (p i).fg) :
  (submodule.pi set.univ p).fg
```
is a more general statement.

Reply:
Oh you're quite right, that was careless of me